charts/backingservices/charts/constellation/README.md

```diff
@@ -1,26 +1,73 @@
-# Constellationui helm chart
+# Constellation UI
```

Comment:
@anuds20 , this description here must start with a link to our primary doc: https://documents.constellation.pega.io/static/88/introduction.html . It should not be duplicating content from that doc to here.

Only have material that is specific to this install here.